src/Billing/Billing/Cmdlets/BillingAccounts/GetAzureRmBillingAccount.cs

```diff
@@ -0,0 +1,107 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Billing.Common;
+using Microsoft.Azure.Commands.Billing.Models;
+using Microsoft.Azure.Management.Billing;
+using Microsoft.Azure.Management.Billing.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Billing.Cmdlets.BillingAccounts
+{
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "BillingAccount", DefaultParameterSetName = Constants.ParameterSetNames.ListParameterSet), OutputType(typeof(PSBillingAccount))]
+    public class GetAzureRmBillingAccount : AzureBillingCmdletBase
+    {
+        private const string AddressExpand = "soldTo";
+
+        const string BillingProfilesExpand = "billingProfiles";
+
+        const string InvoiceSectionsExpand = "billingProfiles/invoiceSections";
+        
+        [Parameter(Mandatory = true, HelpMessage = "Name of a specific billing account to get.", ParameterSetName = Constants.ParameterSetNames.SingleItemParameterSet)]
+        [ValidateNotNullOrEmpty]
+        public List<string> Name { get; set; }
+        
+        [Parameter(Mandatory = false, HelpMessage = "Populate the address for this billing accounts.")]
+        public SwitchParameter IncludeAddress { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "Expand the billing profiles under the billing accounts.")]
+        public SwitchParameter ExpandBillingProfiles { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "Expand the billing profiles and invoice sections under the billing profiles.")]
+        public SwitchParameter ExpandInvoiceSections { get; set; }
```

Comment:
use "ExpandInvoiceSections" instead of "ExpandInvoiceSections"

Reply:
Sure! Done. Using "ExpandInvoiceSection".